.github/workflows/swift-toolchain.yml

```diff
@@ -662,7 +662,7 @@ jobs:
           arch: ${{ matrix.arch }}
 
       - name: Install Swift Toolchain
-        uses: compnerd/gha-setup-swift@8f43ccc3e8bac89829862af09de9567c807c1c12
+        uses: compnerd/gha-setup-swift@main
```

Comment:
Whoops, I got confused looking at the diffbase in our fork. I'll change this back

Reply:
The only difference should be the defaults, nothing else.

Reply:
Reverted.